### ISSUE TITLE
[BUG] Text leaking in net box

### ISSUE THREAD

**Issue by ErrorNoInternet** [aristocratos/btop#522]
**Describe the bug**
Some text (likely from a Cockpit configuration file) is leaking into the net box?

**To Reproduce**
I had just installed Fedora Server aarch64 on my Raspberry Pi and opened btop for the first time.

**Expected behavior**
A clean net box without any text

**Screenshots**
![image](https://user-images.githubusercontent.com/61157623/235605548-380238ca-7f02-46ed-925b-7b141218ffaf.png)

**Info (please complete the following information):**
 - btop++ version: 1.2.13
 - Binary: installed from dnf
 - Architecture: aarch64
 - Platform: Linux
 - (Linux) Kernel: `6.2.9-300.fc38.aarch64`
 - Terminal used: konsole 22.12.3
 - Font used: JetBrains Mono Nerd Font

**Additional context**
```
2023/05/02 (15:30:15) | ===> btop++ v.1.2.13
2023/05/02 (15:30:15) | DEBUG: Starting in DEBUG mode!
2023/05/02 (15:30:15) | INFO: Logger set to DEBUG
2023/05/02 (15:30:15) | DEBUG: Using locale en_US.UTF-8
2023/05/02 (15:30:15) | INFO: Running on /dev/pts/0
2023/05/02 (15:30:15) | DEBUG: Loading theme file: /usr/share/btop/themes/dracula.theme
2023/05/02 (15:30:18) | INFO: Quitting! Runtime: 00:00:03
```

**Comment by Gamrin** [comment]
I'd like to corroborate and say I also experience this.

**Comment by imwints** [comment]
Is this happening every time and can you reproduce that reliably?

**Comment by NorbiPeti** [comment]
Yes, it's coming from the `proc` box. If I scroll that box the extra text scrolls with it and if I remove `proc` then the text goes away.
